### PR TITLE
Math Magicians: Full Website

### PR DESCRIPTION
**Pull request by Munsa1**
# Full Website Using Routes

In the following milestone, I:

- Add my own styles to improve the look and feel.
- Use React basic styling.

### For every page, I create a route and:

- Make sure that navigation links for all 3 routes are displayed on each page.
- Use React Router to make the routes work.

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello @Munsa1 

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+## Copyright 2021, Daniel M. Matongo
+
+
+Permission is hereby granted, free of charge, to any person obtaining a copy of this website and associated documentation files, to deal in the website without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the website, and to permit persons to whom the website is furnished to do so, subject to the following conditions:
```
Comment (mikenath223) [inline MIT.md:4]: - [ ] On the readme the live link there doesn't show this project. Kindly update it.

Comment (Munsa1) [reply MIT.md:4]: Thanks its been handled

**Review by hiromataba — APPROVED** [review]
# Status: Approved :champagne:  :+1: :confetti_ball: 
Hi @Munsa1   :wave: , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### To Highlight :1st_place_medal: 
- Your Pull Request has a proper title and description. ✔️  
- No Linter errors :heavy_check_mark: 
- Your Design looks awesome, keep rocking :rocket: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: MIT.md (hunk 1 above)
Comment (hiromataba) [reply MIT.md:4]: - [ ] [OPTIONAL] Kindly update your live demo or check your deployment

**Pull request merged**